index.html

```diff
@@ -56,7 +56,18 @@ <h2>Derek Dunwiddie</h2>
             <p>on <a href="https://www.linkedin.com/in/derekdunwiddie/">LinkedIn</a></p>
         </div>
         <div>NINE</div>
-        <div>TEN</div>
+        <div>
+            <h1>Mike Gregory</h1>
+                <img src="http://static.rogerebert.com/uploads/movie/movie_poster/the-brotherhood-of-the-wolf-2002/large_1OyUsDSFayOH9LACASkuXOZqkPy.jpg">
```

Comment:
Make sure that you aren't getting a `mixed-content` warning here (try `https` instead of `http`) 😄